### ISSUE TITLE
Show local and regional network cycle connections

### ISSUE THREAD

**Issue by lumica89** [cyclosm/cyclosm-cartocss-style#482]
Hi, 
I have the following feature request based on recent changes in the German OSM Wiki:

Currently the map is mainly rendering bicycle routes and dedicated bicycle infrastructure. A bicycle route is here used for tourism to mark a specific route for a dedicated topic like rivers etc. The map already renders colors according to regional, local etc. and the abbreviations for those routes. That's perfect so far. 
Next to those routes there are also bicycle connections existing that are not part of a specific route and are more used to provide a bicycle connection between cities and villages. Those connections can be based on roads, but also on bicycle infrastructure and are continously signposted.

What I would request as a feature is to show also those connections within the map following the color scheme of the routes (regional, local etc.), with the difference that those connections are tagged differently and have no specific names.

How are those connections be tagged?

1. Option:
On each of the edges is the tag lcn = yes or rcn = yes. lcn stands for local networks, rcn for regional networks.

2. Option
Each of the edges is assigned to a relation of the type "network". This relation has the following attributes:
type = network
route = bicycle
network = rcn | lcn

According to either lcn = yes / rcn = yes or network = rcn / network = lcn the map should show those edges in the colors of regional or local bike routes with the difference that those connections would never have a specific name that should be shown.

**Example in OSM for Option 1:** 
https://www.cyclosm.org/#map=19/50.94060/11.74764/cyclosm 
https://www.openstreetmap.org/way/371254155

**Example in OSM for Option 2:** 
https://www.cyclosm.org/#map=18/50.87064/11.61098/cyclosm 
https://www.openstreetmap.org/way/290098993

**References in German OSM Wiki:**
https://wiki.openstreetmap.org/wiki/DE:Relation:network#Radverkehrsnetze_kartieren 
https://wiki.openstreetmap.org/wiki/DE:Bicycle/Radverkehrsnetze_kartieren 

Thanks for considering this change in advance!

**Comment by Phyks** [comment]
Hi,

Thanks for the bug report!

I think second option should already be rendered since it is basically just a cycle route with an extra tag. 

First option should also be rendered, see https://github.com/cyclosm/cyclosm-cartocss-style/issues/245 although limited to lcn only.

**Comment by lumica89** [comment]
Hi @Phyks , 
I looked for my comments before but I wasn't able to find it (#245). 

Regarding Option 1: 
Yes, you are right. After our discussion in May that should be working. I can see something already for my example, but I'll look on it tomorrow again, when the map was refreshed. For rcn this rendering is still missing.

Regading Option 2:
It is not just a cycle route with an extra tag. A cycle route is defined by a relation with the  type = route, the cycle network connection is defined yb a relation with type = network. So this is not yet rendered as I can see on the cyclosm.

**Comment by Phyks** [comment]
Indeed, I read too fast for option 2. This one indeed needs a bit of work.

**Comment by lumica89** [comment]
I just wanted to add that the German community is discussing the new mapping procedure again since they recognized that the network relations are not yet supported by the OSM cycling map anymore. So please hold on with the implementation of this ticket, I'll come back as soon as I have new information regarding the mapping procedure of networks.
